client/src/Form/Report/DecisionExistType.php

```diff
@@ -14,10 +15,14 @@ class DecisionExistType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('hasDecisions', FormTypes\ChoiceType::class, [
-                'choices' => ['Yes' => 'yes', 'No' => 'no'],
+            ->add('significantDecisionsMade', FormTypes\ChoiceType::class, [
+                // keep in sync with API model constants
+                'choices' => [
+                    'existPage.form.choices.yes' => Report::SIGNIFICANT_DECISION_MADE,
```

Comment:
Noticed when we were pairing that the form wasn't reflecting the translation files properly. Could have been a caching issue. 

Just check it looks fine on the branch environment, and you can compare this form type with others to see if the code is right.

Reply:
All looks fine